### PR TITLE
Adding valve pallet

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: runtime/neumann/src/lib.rs [hunk 1]
```diff
@@ -720,20 +738,23 @@ construct_runtime!(
 		Aura: pallet_aura::{Pallet, Storage, Config<T>} = 23,
 		AuraExt: cumulus_pallet_aura_ext::{Pallet, Storage, Config} = 24,
 
+		// Utilities
+		Valve: pallet_valve::{Pallet, Call, Config, Storage, Event} = 30,
```
Comment (rhuttman) [inline runtime/neumann/src/lib.rs:742]: I am not sure if shifting these will force me to change the version
https://github.com/OAK-Foundation/OAK-blockchain/blob/master/runtime/neumann/src/lib.rs#L183

Comment (irsal) [reply runtime/neumann/src/lib.rs:742]: We'll do a blanket bump before the next upgrade (polka client 0.9.16)

File: pallets/valve/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,144 @@
+// This file is part of OAK Blockchain.
+
+// Copyright (C) 2021 OAK Network
```
Comment (irsal) [inline pallets/valve/src/lib.rs:3]: 2022

File: runtime/neumann/src/lib.rs [hunk 3]
```diff
@@ -694,6 +694,24 @@ impl pallet_automation_time::Config for Runtime {
...
 }
 
+pub struct ClosedCallFilter;
+impl Contains<Call> for ClosedCallFilter {
+	fn contains(c: &Call) -> bool {
+		match c {
+			Call::AutomationTime(_) => false,
+			Call::Balances(_) => false,
+			Call::Bounties(_) => false,
+			Call::Treasury(_) => false,
```
Comment (irsal) [inline runtime/neumann/src/lib.rs:704]: Would consider adding `CollatorSelection`

Comment (irsal) [reply runtime/neumann/src/lib.rs:704]: and `TransactionPayment`

**Review by irsal — CHANGES_REQUESTED** [review]

**Review by irsal — APPROVED** [review]
⛲

**Comment by irsal** [comment]
build is sad

**Pull request merged**